packages/qwik/src/optimizer/src/plugins/plugin.ts

```diff
@@ -875,8 +875,38 @@ export const manifest = ${JSON.stringify(manifest)};\n`;
   // order by discovery time, so that related segments are more likely to group together
   function manualChunks(id: string, { getModuleInfo }: Rollup.ManualChunkMeta) {
     const module = getModuleInfo(id)!;
-    const segment = module.meta.segment as SegmentAnalysis | undefined;
-    return segment?.entry;
+    const segment = module.meta.segment;
+
+    const optimizer = getOptimizer();
```

Comment:
comment here that it's a fallback that avoids bundling too many things together

Reply:
I think it's more about preventing rollup to bundle unrelated chunks together, but I'll add a comment 👍